File: anagramm_test.py
import unittest
import anagramm as an

class Test(unittest.TestCase):
  def test_anagramm(self):
    ''' testing anagramm finction'''
    self.assertTrue(an.anagramm('hello', 'olleh'))
    self.assertFalse(an.anagramm('hello', 'helle'))

File: anagramm_test.go
package coding101class

import (
	"testing"
	"coding101class"
)

func TestAnagramm (t *testing.T){
	got := coding101class.Anagramm("hello", "olleh")
	if !got {
		t.Error("got fail for testing anagramm func")
	}
}